Detectors/ITSMFT/MFT/condition/include/MFTCondition/DCSConfigUtils.h

```diff
@@ -47,34 +48,36 @@ class DCSConfigUtils
   {
   }
 
-  auto getAddress(const std::string name, const std::string type)
+  int getAddress(std::string name, std::string type)
   {
-    return static_cast<int>((mNameDictType[type])[name]);
+    std::map<std::string, int> dict = mNameDict[type];
+    return static_cast<int>(dict[name]);
```

Comment:
why do you need to create a copy of `mNameDict[type]` to query the "name"? Use a reference or do directly
```
return (mNameDict[type])[name];
```

Reply:
DONE.